translations/stripes-smart-components/en.json

```diff
@@ -112,7 +113,16 @@
   "system": "System",
   "notes": "Notes",
   "notes.notFound": "No notes found",
+  "notes.assignNote": "Assign note",
+  "notes.noteSearch": "Note search",
+  "notes.noteAssignmentStatus": "Note assignment status",
+  "notes.found": "{quantity} notes found",
   "date": "Date",
   "title": "Title",
-  "updatedBy": "Updated by"
+  "updatedBy": "Updated by",
+  "status": "Status",
+  "cancel": "Cancel",
+  "saveAndClose": "Save&Close",
```

Comment:
At the least, this should have spaces around the ampersand, but a better solution would probably be to just drop this key and just use the `stripes-core.button.saveAndClose` translation.

Reply:
(There's also a `stripes-core.button.cancel` key as well if you want to drop the `cancel` one you defined.)